docs/sources/administration/correlations/add-permissions-to-create-new-correlations/index.md

```diff
@@ -17,6 +17,6 @@ Adding access to create correlations for [Viewers and Editors]({{< relref "../..
 ## Add permissions to create correlations
 
 1. Go to the Administration section in Grafana.
-1. Open Users page.
+1. Under Users and access, open the Users page.
```

Comment:
nit: I think this should use bold around the UI text but I see the rest of the list doesn't do that currently so it's not a nit for this patch as much as the document itself.